### DISCUSSION TITLE
kafka-ui-api

### DISCUSSION THREAD

**Discussion by tkv29** [provectus/kafka-ui#2227] [Q&A] [answered]
I want to build the docker image for [kafka-ui-api](https://github.com/provectus/kafka-ui/blob/v0.4.0/kafka-ui-api/Dockerfile).

I get following error:
![image](https://user-images.githubusercontent.com/73845255/176864388-ac8c782b-6a54-4a4f-a310-5a493db69adf.png)

Where I can find the needed file?

**Comment by Haarolean** [comment, marked as answer]
Hi, thanks for the interest!

You can either download the release one (every major [release](https://github.com/provectus/kafka-ui/releases) has a jar attached) or you can build one yourself with [maven](https://github.com/provectus/kafka-ui/blob/master/documentation/project/contributing/building.md).

**Reply by tkv29** [reply]
Thank you for your reply!
It says that the jar is invalid or corrupted, any idea?
![image](https://user-images.githubusercontent.com/73845255/176961013-9bdecd1d-c6e2-477f-a9a8-e5fcfc09837a.png)

**Reply by tkv29** [reply]
@Haarolean its strange, I just executed the jar with intellij and it works fine
not sure why it doesnt work in the container

**Reply by Haarolean** [reply]
I feel like the jar hasn't been included upon image build. Check if it does indeed exists where it should.
